cmake/std/PullRequestLinuxCudaDriver.sh

```diff
@@ -15,7 +15,6 @@ set -x
 
 #TODO: review appropriate job size
 #cuda 10.1.105 will be built with rhel7G
-export TRILINOS_PR_DO_NOT_SET_PROXY=TRUE
```

Comment:
@fryeguy52 could you give this an official review to merge in?
This is the only line that I'm not sure if it should be removed or not.
One of the merges may or may not have gone correctly. I just want to make sure with you.
Otherwise, all the Weaver tests are passing.